classroom-dashboard/app/controllers/sections_controller.rb

```diff
@@ -2,6 +2,8 @@ class SectionsController < ApplicationController
 
 	# Find the section before any other action
 	before_action :find_section, only: [:show, :edit, :update, :destroy, :flashcards]
+	
+	 # Do empty methods need to be filled?
```

Comment:
Good point-- I'm not sure if the empty methods are needed. We could try removing them and then seeing if everything works as desired without them?